hack/update-deps.sh

```diff
@@ -25,6 +25,8 @@ cd ${REPO_ROOT_DIR}
 # Ensure we have everything we need under vendor/
 dep ensure
 
+check_licenses $(go list ./...)
```

Comment:
I don't think this is particularly fast (not that `dep` is), and we already run it in presubmit tests...

Reply:
Took it out of here for the presubmit test to catch.